chingsley/ireporter-react
# #165660137 - Style home page and ensure all pages contain the mvp components

#### What does this PR do?
It styles the home page and ensures all pages contain the MVP components

#### Description of Task to be completed?
- Style the home page
- Create all needed pages
- Render all the required components on the page

#### How should this be manually tested?
-pull the branch
- ensure all dependencies are installed, by running `npm i`
- start the app by running `npm start:dev`
- check out the pages by clicking the links on the header
- Create a new article and have someone else comment on or like the article. You will get an email 

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/165660137

#### Screenshots (if appropriate)
- N/A

#### Questions:
N/A
